assets/js/src/delete-term-meta.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Add select2 functionality..
+ */
+ /*global ajaxurl*/
+jQuery( document ).ready( function () {
+	
+	jQuery( 'input[name=smbd_meta_term_taxonomy]' ).change(function(){
+		jQuery( '.select2-terms' ).select2( {
```

Comment:
Don't use the class name `select2`. It is causing issues.

Rename it to `enhanced-terms-dropdow`

Reply:
Fixed